### PR TITLE
refactor query

### PR DESCRIPTION
**Pull request by ttt-77**
Solve issue #116 . Previous code return a query expression when changing original query, now return a new query with type Query.

### CODE REVIEW

**Comment by ddkang** [comment]
@akash17mittal @continue-revolution can you check this?

File: requirements.txt [hunk 1]
```diff
@@ -37,7 +37,7 @@ six==1.16.0
 sniffio==1.3.0
 SQLAlchemy==1.4.39
 SQLAlchemy-Utils==0.41.1
-sqlglot-aidb==0.0.7
+sqlglot-aidb==0.0.9
```
Comment (ddkang) [inline requirements.txt:40]: Oops why was this jumped by 2?

**Comment by ddkang** [comment]
Overall looks okay to me

File: requirements.txt (hunk 1 above)
Comment (ttt-77) [reply requirements.txt:40]: 0.0.8 is supporting user function, 0.0.9 is supporting where rewriter and join rewriter

**Review by ddkang — APPROVED** [review]

**Pull request merged**